### ISSUE TITLE
GetInvoiceAsPdf() fails

### ISSUE THREAD

**Issue by channeladam** [XeroAPI/Xero-NetStandard#104]
The documentation states:

> Individual Invoices and Credit Notes can also be returned in PDF format by setting the “Accept” value in the http header to “application/pdf”.

The AccountingApi.cs code however attempts to call a URL of /Invoices/{InvoiceID}/pdf with an Accept of application/octet-stream.

> `var response = await this.AsynchronousClient.GetAsync<System.IO.Stream>("/Invoices/{InvoiceID}/pdf", requestOptions, this.Configuration);`

Same with credit notes it appears.

The /Invoices/{InvoiceID}/pdf URL makes a 404 happen when I call the URL manually through Postman.
However, if in Postman I call /Invoices/{InvoiceID} with Accept application/pdf, it works.

Also, the C# method signature is weird, containing a contentType parameter which doesn't make sense given that the method name is GetInvoiceAsPDF().

**Comment by SidneyAllen** [comment]
@channeladam - I've updated our mustache template and OpenAPI specs to address this issue. It should be fixed in our next release by @jenksguo 

OpenAPI spec release notes
https://github.com/XeroAPI/Xero-OpenAPI/releases/tag/2.0.1

**Comment by SidneyAllen** [comment]
@channeladam - We have updated our OpenAPI spec to use the correct path.

We still need to test this in the SDK, but will update shortly.

**Comment by jenksguo** [comment]
@channeladam This is now addressed in the new version of SDK v0.4.3. 

Please see following implementation in a MVC example: 

```

     Guid invoiceId = Guid.Parse("d5654fc7-******-802a-9f5ab9d2476a");

      var AccountingApi = new AccountingApi();

      var response = await AccountingApi.GetInvoiceAsPdfAsync(accessToken, xeroTenantId, invoiceId);

      return new FileStreamResult(response, "application/pdf");
```
